src/Framework/Runner/Interfaces.cs

```diff
@@ -85,6 +85,7 @@ public interface IRunnerSetupData
         bool IsTesting { get; set; }
         string ExcludedCategory { get; set; }
         int SelectedProduct { get; set; }
+        string AdditionalResolutionDirectories { get; set; }
```

Comment:
Ok, I see that you mean to pass multiple directories in one string separated by ";".